.github/workflows/industrial_ci_action.yml

```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  ADDITIONAL_DEBS: "clang-tidy libclang-dev"
+  CATKIN_LINT: true
+  CATKIN_LINT_ARGS: '--strict'
+  CMAKE_ARGS: "-DCATKIN_ENABLE_CLANG_TIDY=true"
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        env:
+          - {ROS_DISTRO: melodic, ROS_REPO: testing}
```

Comment:
I suppose we will activate `noetic` in a future PR?

Reply:
Yes.